### PR TITLE
Introduce pathname patterns to CLI 

### PR DESCRIPTION
**Pull request by junxiant**
I commented out the argument using individual images and added an argument to use folder directory  containing images instead.

### CODE REVIEW

File: stitching/cli/stitch.py [hunk 1]
```diff
@@ -251,7 +259,26 @@ def main():
...
+    list = []
+    for extension in extensions:
+        list.extend(glob.glob(image_paths+"/"+extension))
+
+    # Get all images
+    img_names = []
+    for image in list:
+        # img = cv2.imread(image)
+        img_names.append(image)
+    #####
```
Comment (lukasalexanderweber) [inline stitching/cli/stitch.py:279]: this creates the identical list

File: stitching/cli/stitch.py [hunk 2]
```diff
@@ -5,7 +5,8 @@
 import argparse
 
 import cv2 as cv
-import numpy as np
+import numpy as 
+import glob
```
Comment (lukasalexanderweber) [inline stitching/cli/stitch.py:9]:
```suggestion
import argparse
import glob

import cv2 as cv
import numpy as 
```

**Comment by lukasalexanderweber** [comment]
thanks for your contribution!

**Comment by junxiant** [comment]
Sure man your suggestions seems more flexible

**Pull request merged**